packages/server/package.json

```diff
@@ -7,22 +7,27 @@
   "main": "./src/index.mts",
   "scripts": {
     "build": "tsc && tsc-alias",
+    "build-subgraph-schema": "tsx scripts/generate-subgraph-schema.ts",
     "dev": "nodemon --exec \"yarn start | pino-pretty\" --ignore dist --watch 'src/**/*' -e ts,tsx,mts",
     "generate": "yarn generate:all",
     "generate:sql": "dotenv -e ../../.env yarn pgtyped",
     "generate:sql:watch": "dotenv -e ../../.env yarn pgtyped:watch",
     "pgtyped": "yarn rimraf **/__generated__/*.types.ts; DATABASE_URL=$(node scripts/set-db-url.cjs) pgtyped -c $(node scripts/generate-local-pgconfig.cjs)",
     "pgtyped:watch": "yarn pgtyped -w",
+    "prepgtyped": "rimraf **/__generated__/*.types.ts",
     "server:dev": "yarn dev",
     "start": "yarn build && node dist/server/src/index.js"
   },
   "dependencies": {
     "@accounter/green-invoice-graphql": "workspace:^",
     "@accounter/pcn874-generator": "workspace:^",
     "@ai-sdk/anthropic": "1.1.13",
+    "@anthropic-ai/sdk": "0.36.3",
```

Comment:
I think `@anthropic-ai/sdk` is leftovers from one of the rebases, and not in use anymore

Please as a final step go over the code and make sure all of those are cleaned (as some more might appear on a future rebase)